.github/workflows/ml-wheels.yml

```diff
@@ -0,0 +1,39 @@
+name: build-many-linux-wheels
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+
+jobs:
+  ml-wheels:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+    
+      - name: upgrade pip
+        run: |
+          python -m pip install --upgrade pip
+
+      - name: Build manylinux Python wheels
+        uses: RalfG/python-wheels-manylinux-build@v0.4.2-manylinux2014_x86_64
+        with:
+          python-versions: 'cp37-cp37m cp38-cp38 cp39-cp39 cp310-cp310'
+          build-requirements: 'cython numpy scipy libsvmdata scikit-learn xarray download'
+
+      - name: publish to Test PyPI
+        env:
+          TWINE_USERNAME: ${{ secrets.TEST_PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_PASSWORD }}
+        run: |
+          twine upload --repository testpypi dist/*-manylinux*.whl
```

Comment:
we can upload everything, no ?

Reply:
I'm not sure. 
Pls, have a look on [this link](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#:~:text=You%20should%20also%20create%20a%20wheel%20for%20your%20project.%20A%20wheel%20is%20a%20built%20package%20that%20can%20be%20installed%20without%20needing%20to%20go%20through%20the%20%E2%80%9Cbuild%E2%80%9D%20process.%20Installing%20wheels%20is%20substantially%20faster%20for%20the%20end%20user%20than%20installing%20from%20a%20source%20distribution.)